cli/src/factory_ops/unboxing/network/bmc

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+set -euE
+export NETWORK_BMC=1
+
+LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/unboxing_nw_bmc.log}"
+mkdir -p $(dirname "${LOG_FILE}")
+#truncate -s 0 ${LOG_FILE}
+
+PRVSNR_ROOT="/opt/seagate/cortx/provisioner"
+SALT_OPTS="--no-color --out-file=${LOG_FILE} --out-file-append"
+
+pvt_ip_a=$(get_pillar_data cluster:srvnode-1:network:data_nw:pvt_ip_addr)
+pvt_ip_b=$(get_pillar_data cluster:srvnode-2:network:data_nw:pvt_ip_addr)
+
+remotes=(
+    ${pvt_ip_a}
+    ${pvt_ip_b}
+)
+
+# Set BMC dhcp network configs
+function set_bmc_dhcp_config {
+    _linfo "================================================================================"
+    _linfo "Setting BMC network information to DHCP on both nodes"
+    _linfo "================================================================================"
+    _linfo "    "
+    _linfo "You have selected to proceed with DHCP based configuration for management network interface."
+    _linfo "    "
+    proceed_check
+    _linfo "    "
+
+
+    for remote in ${remotes[@]}; do
+        _linfo "Setting BMC Network Information for DHCP on ${remote}"
+
+        ssh_over_pvt_data ${remote} "ipmitool lan set 1 ipsrc dhcp"
+        ssh_over_pvt_data ${remote} "ipmitool lan set 1 access on"
+        _linfo "Sleeping for 30 secs for the settings to take effect"
+        ssh_over_pvt_data ${remote} "sleep 30"
```

Comment:
why do we need ssh here?

Reply:
It unifies the logic.
We need to configure both nodes, for node-2 we need ssh.
Generically making both calls over ssh doesn't harm, so I've used the same logic for both nodes.